internal/controller/promotion/helm_test.go

```diff
@@ -321,8 +321,8 @@ func TestBuildValuesFilesChanges(t *testing.T) {
 		t,
 		map[string]map[string]string{
 			"fake-values.yaml": {
-				"fake-key":        "fake-url:fake-tag",
-				"second-fake-key": "second-fake-tag",
+				"fake-key":        "fake-url:'fake-tag'",
```

Comment:
This test case demonstrates the problem I mentioned [here](https://github.com/akuity/kargo/pull/1315/files#r1434538580).

`fake-url:'fake-tag'` isn't a valid image identifier.

Reply:
@krancour thank you, I fixed it https://github.com/akuity/kargo/pull/1315/commits/df36c0f753b648d7bb18953b637d694642200d42
I still had to keep single quotes in the tagOnly test case as otherwise it fails, is it OK?